src/ios/CDVCamera.m

```diff
@@ -344,49 +429,6 @@ - (void)popoverControllerDidDismissPopover:(id)popoverController
     self.hasPendingOperation = NO;
 }
 
-- (NSData*)processImage:(UIImage*)image info:(NSDictionary*)info options:(CDVPictureOptions*)options
```

Comment:
[Just so I can remember to check what replaced the functionality that used to be provided by this funtion]